googlemaps/client.py

```diff
@@ -272,6 +287,10 @@ def _generate_auth_url(self, path, params, accepts_clientid):
 from googlemaps.roads import snap_to_roads
 from googlemaps.roads import speed_limits
 from googlemaps.roads import snapped_speed_limits
+from googlemaps.places import places
+from googlemaps.places import place
+from googlemaps.places import photo
+from googlemaps.places import autocomplete
```

Comment:
Fair point, I've renamed `photos` and `autocomplete` in ea670bcf5a0703a05ecc5167300f9303d2c81253